src/app/settings/user-subscription.component.ts

```diff
@@ -210,4 +210,8 @@ export class UserSubscriptionComponent implements OnInit {
   get usingInAppPurchase() {
     return this.sub != null ? this.sub.usingInAppPurchase : false;
   }
+
+  get title(): string {
+    return this.i18nService.t(this.selfHosted ? "subscription" : "premiumMembership");
```

Comment:
Is this in the wireframes? I don't understand why we'd change the title this way.

Reply:
Good catch: It wasn't originally but the wireframes do have a header now. For the `Self Hosted` flow it shows `Subscription` instead of the current `Premium Membership` - I will double check with design today, since I technically did the headers before they were added into the wireframes. 👍

Reply:
I talked with design and the wireframes have been updated to reflect what is currently coded. This should be good to review 👍